packages/terra-dropdown-button/tests/wdio/split-button-spec.js

```diff
@@ -331,6 +331,26 @@ Terra.describeViewports('Split Button', ['medium'], () => {
     });
   });
 
+  describe('With icon', () => {
+    it('displays a button with icon first', () => {
+      browser.url('/raw/tests/cerner-terra-core-docs/dropdown-button/left-icon-split-button');
```

Comment:
All these example seem pretty small, so I wonder if we can optimize these tests by having them all on the same page. Unless if it's not worth the effort.

Reply:
sure, combined them into one test/snapshot here: [7396391](https://github.com/cerner/terra-core/pull/4080/commits/73963913e58237f8cedd81c14e5bef5df5e2b5e9)

Reply:
That looks good! Can you also add text labels for easier identification? 

e.g.
```jsx
<h3>icon left</h3>
<iconLeftExample

<h3>icon right</h3>
<iconRightexample
```

Reply:
Updated: [f66ce00](https://github.com/cerner/terra-core/pull/4080/commits/f66ce0032598bde290504304cf77b268ad95433d)